README.rst

```diff
@@ -0,0 +1,43 @@
+=====================
+MOJ Product Dashboard
+=====================
+
```

Comment:
Can you put the code climate badges in to the readme - https://codeclimate.com/github/ministryofjustice/moj-product-dashboard